Container/Containerfile

```diff
@@ -51,6 +51,7 @@ WORKDIR /
 

```

Comment:
It looks like the container build is failing at the moment, I think adding this line should fix things?

```suggestion

RUN apt-get install -y pkg-config libhdf5-dev

```